backend/routes/shops.js

```diff
@@ -800,6 +800,7 @@ module.exports = function (router) {
         'offlinePaymentMethods',
         'supportEmail',
         'upholdClient',
+        'useEscrow',
```

Comment:
Initially when we deploy this PR, no merchant will have this config set. Which means it will default to undefined false. And therefore the marketplace won't be used for purchases on their shop.

The alternative would be to set useEscrow to true on all existing shops using a script. Not worth it in my opinion, since I haven't heard any merchants saying they want escrow. But just mentioning it...

Reply:
Yea I think we default to not using escrow even for existing shops.

For 475 perhaps we can use a network level config for deciding whether to record off-chain payments to the marketplace contract? It should probably apply globally rather than per shop